src/data/player.py

```diff
@@ -312,7 +312,9 @@ def add_points(self, points: float):
             self.points += points
 
     def to_trf(
-        self, player_id_to_trf_id: Callable[[int], int], max_round: int,
+        self,
+        player_id_to_trf_id: Callable[[int], int],
+        after_round: int,
```

Comment:
Make `after_round` keyword-only to avoid ambiguity.
Maybe `player_id_to_trf_id` should be not be a positional-only argument, however.
```suggestion
        self,
        player_id_to_trf_id: Callable[[int], int],
        /,
        *,
        after_round: int,
```